doc/TorEvents.markdown

```diff
@@ -0,0 +1,301 @@
+# Tor Events used by PrivCount
+
+PrivCount obtains its Tor usage data via Tor Control Events. These events have
+been implemented specifically for PrivCount. PrivCount also uses some other
+Controller interfaces.
+
+## PrivCount Event Overview
+
+### Initialisation
+
+#### PrivCount authenticates to the Control Port (PROTOCOLINFO, AUTHENTICATE),
+     using one of the following methods:
```

Comment:
Since this is a markdown file, be careful with formatting. Adding spaces to the beginning of the line, like in front of "using" in this case, actually starts a code block. In general, issues like this will be noticeable when viewing the file on GitHub.

(There are additional instances of this bug in this file, but I won't repeat this comment.)